packages/client/src/pages/user-info.vue

```diff
@@ -54,6 +54,9 @@
 				<FormButton v-if="user.host != null" class="_formBlock" @click="updateRemoteUser"><i class="fas fa-sync"></i> {{ $ts.updateRemoteUser }}</FormButton>
 			</FormSection>
 
+			<MkObjectView v-if="info && info.email !== undefined" tall :value="info">
```

Comment:
I think checking for `info.email` here is a suboptimal solution, not least because some instances do not require email addresses for signup. I checked on my own instance and in such a case `info.email` is not set so would be `undefined`.
I'm not sure if the `v-if` is needed here at all.

Reply:
v-if is used to avoid showing the following data which is entirely redundant to moderators
```ts
return {
    isModerator: user.isModerator,
    isSilenced: user.isSilenced,
    isSuspended: user.isSuspended,
};
```